install_config/upgrading/os_upgrades.adoc

```diff
@@ -0,0 +1,27 @@
+[[install-config-upgrading-os-upgrades]]
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+When you need to upgrade the Operating System, either changing OS
+versions or just updates to the system software, you need to be aware
+that this can have an impact on the {product-title} software running
+on those machines.
+
+In particular, these updates can affect the `iptables` rules that
```

Comment:
But we own OVS and update the rules.  The fundamental problem is that docker doesn't monitor and refresh its iptables rules... so if you get lucky and you snapshotted the handful of docker rules that matter, then things will work when you restart because openshift will fix the rest of the rules.

Anyway, I'll call ot ovs too.